charts/helmet/templates/_deployment.yaml

```diff
@@ -55,7 +55,7 @@ spec:
           securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
           {{- end }}
           image: {{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          imagePullPolicy: {{ default (eq .Values.image.tag "latest" | ternary "Always" "IfNotPresent") .Values.image.pullPolicy }}
```

Comment:
Regardless, we prefer not to change this part because Kubernetes handles the image pulling behavior in a specific way. Please review this section of the Kubernetes documentation: [Default image pull policy](https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting).

Reply:
Thanks for the review! I know how pull policies work, but I reviewed the documentation anyway in case I've missed something.

What's happening here is since helmet unconditionally renders this value into the manifest _**we don't allow Kubernetes' default pull policy logic to kick in**_. The documentation you linked clearly states that the kubernetes default only applies ["if you omit the `imagePullPolicy` field"](https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting:~:text=conditions%20are%20met%3A-,if%20you%20omit%20the%20imagePullPolicy%20field,-%2C%20and%20you%20specify). 

Since helmet's exported default values set imagePullPolicy to `Always`, even if a user sets a digest or a tag other than latest, the pull policy will still be rendered as `Always`. https://github.com/companyinfo/helm-charts/blob/6bda028afdc3c4c15b225084e7104b17714e77d9/charts/helmet/values.yaml#L76-L80

Reply:
Another way to fix this would be to change the default in helmet's exported default values to `pullPolicy: null` and change the template to something like this, so that the user _can_ omit the imagePullPolicy field if they choose to
```yaml
{{ with .Values.image.pullPolicy }}
imagePullPolicy: {{ . | quote }}
{{ end }}
```

Reply:
@zevisert Please set `pullPolicy` to `null` on values.yaml, and then I can merge the PR.

Reply:
Done! :)